custom_components/tuya_local/devices/vacplus_dehumidifier.yaml

```diff
@@ -0,0 +1,145 @@
+name: Vacplus Dehumidifier
```

Comment:
Leave out the brand name above, as other brands may match the same dps layout.  The brand in the products section ensures it is only applied to the correct product.

Reply:
What do you recommend here? Dehumidifier seems too generic for something like this, as there are already plenty of different Dehumidifier models in this project.